torch/csrc/jit/tensor_conversions.h

```diff
@@ -83,67 +83,9 @@ inline T tensor_as(at::Tensor t) {
   return detail::tensor_as_impl<T>()(std::move(t));
 }
```

Comment:
Do we still need `tensor_as`?

Reply:
I removed it. Turns out it was still being used in a few places which required changing Node::get to look at the schema to distinguish an at::Scalar attribute from an at::Tensor one. Not too worried about this since we are going to remove it as an attribute anyway.